Gems/AudioSystem/Code/Include/Engine/AudioFileUtils.h

```diff
@@ -20,22 +21,26 @@ namespace Audio
     /*!
      * FindFilesInPath
      */
-    static AZStd::vector<AZStd::string> FindFilesInPath(const AZStd::string_view folderPath, const char* filter)
+    static AZStd::vector<AZ::IO::FixedMaxPath> FindFilesInPath(const AZStd::string_view folderPath, const char* filter)
     {
-        AZStd::vector<AZStd::string> foundFiles;
+        AZStd::vector<AZ::IO::FixedMaxPath> foundFiles;
         AZ::IO::FileIOBase::FindFilesCallbackType findFilesCallback = [&foundFiles](const char* file) -> bool
         {
-            foundFiles.emplace_back(file);
+            foundFiles.emplace_back(AZ::IO::FixedMaxPath{ file }.LexicallyNormal());
```

Comment:
To avoid constructing an unnecessary FixedMaxPath, a PathView can be used instead for the `LexicallyNormal` call.

Reply:
Will change it to PathView.